login-fire-button.html

```diff
@@ -198,13 +198,13 @@
      * The provider logo must be hidden if the element can sign out and a user
      * is currently signed in.
      *
-     * @param  {Boolean} onlySignIn is `false` if the element can sign out
+     * @param  {Boolean} noSignOut is `false` if the element can sign out
      * @param  {Boolean} signedIn   is `true` if a user is actually signed in
      * @return {Boolean}            `true` if the element can sign out and a
      * user is currently signed in.
      */
-    _computeHideIcon: function(onlySignOut, onlySignIn, signedIn) {
-      return onlySignOut || (!onlySignIn && signedIn);
+    _computeHideIcon: function(noSignIn, noSignOut, signedIn) {
+      return noSignIn || (!noSignOut && signedIn);
```

Comment:
Starting over with this. This calculates whether a provider icon is shown on the button so, it should show the provider icon in one of these scenarios:
1- you can sign in and you are not signed in = `!noSignIn && !signedIn`
2- you can't sign out = `noSignOut`

So the end result is `(!noSignIn && !signedIn) || noSignOut`.

Does that sound right or is there a case I'm missing?